server-connection/src/test/java/org/sonarsource/sonarlint/core/serverconnection/storage/XodusServerIssueStoreMigrationTests.java

```diff
@@ -30,6 +30,8 @@
 import org.sonarsource.sonarlint.core.commons.log.SonarLintLogTester;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
```

Comment:
These static imports seem to be not used in the changes proposed, please remove them if not used.